dkobo/koboform/views/survey_draft_views.py

```diff
@@ -234,16 +307,27 @@ def publish_survey_draft(request, pk, format=None):
         status_code = 504
 
     if 'formid' in resp:
```

Comment:
the helper can be returned by this function, and the helper function itself can build a `Response` object with the appropriate `resp` and `status`